frappe/patches/v14_0/remove_manage_subscriptions_from_navbar.py

```diff
@@ -0,0 +1,10 @@
+import frappe
+
+
+def execute():
+	navbar_settings = frappe.get_single("Navbar Settings")
+	for i, l in enumerate(navbar_settings.settings_dropdown):
+		if l.item_label == "Manage Subscriptions":
+			navbar_settings.settings_dropdown.pop(i)
```

Comment:
note for future: we can remove a child doc just by doing `doc.remove(child_doc)`